content/v2/certificates.markdown

```diff
@@ -169,19 +169,25 @@ Custom names require a subscription to a Professional or Business plan.
 
 A certificate can be purchased for multiple subdomains. We call them _alternate names_ or [Subject Alternative Name (SAN)](https://support.dnsimple.com/articles/what-is-ssl-san/).
 
-By default a certificate does not have alternate names.
+By default, a certificate doesn't have alternate names.
 
 You can purchase a single certificate for both `https://docs.example.com` and `https://status.example.com`, alongside  `https://example.com`.
 
 Alternate names require a subscription to a Professional or Business plan.
 
+### Wildcard certificates
+
+To request a wildcard certificate that's valid for all subdomains of a given domain, use `*` (e.g. `*.example.com`).
+
+Wildcard certificates require a subscription to a Professional or Business plan.
```

Comment:
Can we use here the same info format of https://github.com/dnsimple/dnsimple-developer/blob/master/content/v2/vanity.markdown#vanity-name-servers-api ?